scss/_eb-reset.scss

```diff
@@ -1,6 +1,6 @@
 // For now the following seems to get the job done. However we may determine that something more agressive like http://cleanslatecss.com might need consideration.
```

Comment:
@dallasbpeters can you also rename this file?

Reply:
done!